app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -173,6 +175,118 @@ private fun SettingHydrationNotificationToggle(
     )
 }
 
+@Composable
+private fun SettingApplicationTheme(
+    model: SettingsViewModel = hiltViewModel()
+) {
+    val themeValue by model
+        .get(Setting.ApplicationTheme)
+        .collectAsStateWithLifecycle()
+
+    val state = rememberPickerState(initialValue = themeValue)
+
+    val valueText = when (themeValue) {
+        1 -> "Dark"
+        2 -> "Light"
+        3 -> "Dynamic"
+        else -> themeValue.toString()
+    }
+
+    SettingCard(
+        description = "Choose the theme of the application. Default is the 'Dynamic Theme'.",
+        editContent = {
+            Column(
+                modifier = Modifier
+                    .clip(shape = MaterialTheme.shapes.small)
+                    .background(
+                        color = MaterialTheme.colorScheme.surfaceContainer,
+                        shape = MaterialTheme.shapes.small
+                    )
+                    .wrapContentSize()
+                    .padding(32.dp)
+            ) {
+                Text(
+                    style = MaterialTheme.typography.titleLarge,
+                    text = "Select Application Theme"
+                )
+
+                Spacer(modifier = Modifier.size(16.dp))
+
+                Row(
```

Comment:
this whole Row could be wrapped in a function to avoid repeating same code three times